core/server/middleware/error-handler.js

```diff
@@ -81,6 +82,8 @@ _private.HTMLErrorRenderer = function HTMLErrorRender(err, req, res, /*jshint un
     // This ensures that no matter what res.render will work here
     if (_.isEmpty(req.app.engines)) {
         req.app.engine('hbs', _private.createHbsEngine());
+        req.app.set('view engine', 'hbs');
+        req.app.set('views', config.get('paths').defaultViews);
```

Comment:
Unrelated to this line.

If we would add `message: err.message` to https://github.com/TryGhost/Ghost/pull/8832/files#diff-6b41d6f0ef148f7d202931a87d5c4130R37, this would prefers the inherited error message. 

![gaga1](https://user-images.githubusercontent.com/1160712/28912710-72c023f2-7846-11e7-96f0-9d5c25898afb.png)
![gaga2](https://user-images.githubusercontent.com/1160712/28912713-742f7b3e-7846-11e7-9f88-ec38a0f42fd7.png)